langchain/evaluation/run_evaluators/string_run_evaluator.py

```diff
@@ -203,7 +206,12 @@ def map(self, example: Example) -> Dict[str, Any]:
                     " specify a reference_key."
                 )
             else:
-                return {"reference": list(example.outputs.values())[0]}
+                output = list(example.outputs.values())[0]
+                return {
+                    "reference": output
+                    if type(output) == str
+                    else self.serialize_chat_messages([output])
```

Comment:
It's a bit presumptuous to assume the example outputs will be either messages or strings right? They can be arbitrary objects? Think we should probably only handle those two cases in this mapper but throw a better error to the user if they try to use this with some crazy dataset

Reply:
Only valid for the LLM/Chat model data type datasets ya

Reply:
I can change the condition. But note the previous condition also assumes the value is stringified

Reply:
I've updated the condition